include/libfirm/iredgekinds.h

```diff
@@ -13,6 +13,7 @@
 #define FIRM_IR_IREDGEKINDS_H
 
 #include "begin.h"
+#include "firm_types.h"
```

Comment:
The `#include "begin.h"` must be the last include.

Please move the new include in front of the `#include "begin.h"` and add an empty line between them.

Reply:
okay done, dont forget to squash as I edit online.